src/schema-gen/mvn-xsd-schema-xsl.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# Fail early if an error occurs
+set -Eeuo pipefail
+
+usage() {
+    cat <<EOF
+Usage: $(basename "${BASH_SOURCE[0]}") METASCHEMA_SOURCE XSD_RESULT [ADDITIONAL_ARGS]
+
+Produces an XSD (XML Schema Definition) from a valid and well-ordered NIST (ITL/CSD) metaschema (Metaschema instance) using Saxon invoked from Maven.
+Please install Maven first.
+
+Additional arguments are provided to SaxonHE - see https://www.saxonica.com/documentation11/#!using-xsl/commandline
+EOF
+}
+
+if ! [ -x "$(command -v mvn)" ]; then
+  echo 'Error: Maven (mvn) is not in the PATH, is it installed?' >&2
+  exit 1
+fi
+
+[[ -z "${1-}" ]] && { echo "Error: METASCHEMA_SOURCE not specified"; usage; exit 1; }
+METASCHEMA_SOURCE=$1
+[[ -z "${2-}" ]] && { echo "Error: XSD_RESULT not specified"; usage; exit 1; }
+XSD_RESULT=$2
+
+ADDITIONAL_ARGS=$(shift 2; echo ${*// /\\ })
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
+POM_FILE="${SCRIPT_DIR}/../../support/pom.xml"
+
+MAIN_CLASS="net.sf.saxon.Transform" # Saxon defined in pom.xml
+
+if [ -e "$XSD_RESULT" ]
+then 
+    echo "Deleting prior $XSD_RESULT ..."
+    rm -f ./$XSD_RESULT
+fi
+
+# mvn  -f ../pom.xml exec:java -Dexec.mainClass="com.xmlcalabash.drivers.Main"  -Dexec.args="-oHTML=test.html -oFO=/dev/null render-oscal-catalog.xpl
+mvn \
```

Comment:
Minor, non-blocking: it would be preferable to `result=$(mvn ...)` to check the exit code is 0 or not 0 to ensure you know there was a successful execution or not, regardless of whether or not a resulting file is created with the following check. Just my two cents.